pyodide_build/pywasmcross.py

```diff
@@ -367,7 +371,11 @@ def calculate_object_exports_readobj(objects: list[str]) -> list[str] | None:
 def calculate_object_exports_nm(objects: list[str]) -> list[str]:
     args = ["emnm", "-j", "--export-symbols"] + objects
     result = subprocess.run(
-        args, encoding="utf8", capture_output=True, env={"PATH": os.environ["PATH"]}
+        args,
+        encoding="utf8",
+        capture_output=True,
+        env={"PATH": os.environ["PATH"]},
+        check=True,
```

Comment:
We check the result ourselves
```suggestion
        check=False
```